### ISSUE TITLE
[CAIA A11y Research - PILOT] Disability Benefits: Contention Classification Team - New Conditions Page: Form: 21-526EZ (July 2024)

### ISSUE THREAD

**Issue by sara-amanda** [department-of-veterans-affairs/va.gov-team#76779]
> [!IMPORTANT]
> - **TICKET STATUS:** `Recommended` Spoke to Team in Kickoff` on 2/21/2024
> - **AUDIT**: `COMPLETED` Audit of Prototype Prior to Research [76766](https://github.com/department-of-veterans-affairs/va.gov-team/issues/76766) ✅ 
> - **PILOT DATE:**  `COMPLETED` 7/22/2024 - prior to the start of research ["early" July 2024](https://github.com/department-of-veterans-affairs/va.gov-team/issues/76707#issuecomment-2050109744)
> - **PREP WORK (INTERNAL-CAIA) - [Prep document for Sara and Jamie](https://docs.google.com/document/d/1aqAMiWe55lQTlnwKzs7VxtKCJGTQOh_TMnO9z4gjj_w/edit#heading=h.7b53y4dqihzk)
> - **TYPES OF ASSISTIVE TECH:** JAWS + WINDOWS (PILOT) ; Testing will use VoiceOver (majority); TalkBack and Screen Magnification (ZoomText). 

# CAIA A11y Support
This is a child ticket of the research ticket [76707](https://github.com/department-of-veterans-affairs/va.gov-team/issues/76707) - created to track work, completed from this pilot session.

> [!TIP]
> **DEFINITION OF DONE**: Refer to the task list in this ticket, for actionable items.

## Two Variations to Test
> "We can test variations of both options or consolidate to one if there's a strong opinion."
### Variations
1. "Removing additional information"
2. "Adding additional information to component". 

## CAIA Assignee(s)

<details><summary>Toggle to view CAIA info</summary>

- Jamie (Pilot)
- Sara (Co-Pilot)

</details>

## Team Info

<details><summary>Toggle to view team info</summary>

- **Team name:** Contention Classification
- **OCTO product owner:** Jen Bertsch
- **Product name:** 526EZ
- **Product manager:** Yurena Garcia
- **Slack channel:** [benefits-contention-classification](https://join.slack.com/share/enQtNjYwMDExOTQyMDc1Ny1iOGNlYTE4ZjY4OGM2YmUzOWE3YmY3MTMwMzA4MzRkMzQyNWVjMzE3NDQ0ODk4NDMzYmJhNjNmNGRiN2FjM2Vj)
- **Dedicated content writer on your team (if you have one):** N/A
- **Dedicated a11y specialist on your team (if you have one):** N/A

</details>

## Description

<details><summary>Toggle to view the description</summary>

A CAIA a11y will participate as a user in a pilot of the [Contention Classification New Conditions Page Form: 21-526EZ research study](https://github.com/department-of-veterans-affairs/va.gov-team/issues/76707).

- **Researcher**: The VFS team researcher will test their prototype and conversation guide by conducting the pilot session with a CAIA A11Y.
- **User**: The CAIA A11Y team member will act as a real user in the pilot session.
- **Assistive Tech**: The CAIA A11Y team member will follow the prompts given by the VFS Researcher, while using and demonstrating how the behavior of a `screen reader` would behave.

### Pilot Session
- **Date:** `TBD` - (Research starts "early" July 2024)
-  [Prototype in CodePen](https://codepen.io/ccteam/live/ZEZeMPR) RCVD 3/22/2024
- ~Staging~ [**Link for Pilot**](URL: https://www.va.gov/disability/file-disability-claim-form-21-526ez/new-disabilities-revised/add) `RCVD 7/19/2024`
- [**Research conversation guide** - RCVD 7/10/2024](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/products/disability/526ez/research/CC-Team-Research/202403%20Improved%20Autosuggest/conversationguide-assistivetech.md)
- [**Research plan**](https://github.com/department-of-veterans-affairs/vagov-claim-classification/blob/main/collab-cycle/%5Bimproved%20autosuggest%20component%5D/research-plan.md)

</details>

### Assistive Tech Options

> [!TIP]
> **DEVICE SUPPORT DURING RESEARCH SESSIONS**
> - **Available AT**: `Screen readers` on a `desktop` device is currently the primary source of AT that CAIA A11ys are able to support, in addition to `screen magnification` for user research sessions.
> - **Additional Specifications:** If there is a particular `test user`, `browser` or `operating system` you would like us to use, please let us know in advance via this ticket's comments, tagging the assignee - or reach out to @coforma-terry in the CAIA Slack Channel: [#sitewide-content-accessibility-ia ](https://dsva.slack.com/archives/C01K37HRUAH)

## Definition of Done

```[tasklist]
### A11y Tasks
- [x] **Schedule CAIA a11y (prior to the start of research in "early" July):** (update assignees in ticket)
- [x] **Determine date and time for pilot (prior to the start of research in "early" July):** July 22
- [x] **Update CAIA A11y Capacity Calendar:** Add invite for pilot, when scheduled and flag who is covering it in the title
- [x] **Review research** [conversation guide](https://docs.google.com/document/d/1xIyVE2QOv4HornhwAn-bhn3nj_MqF_2NAbP5OT_Lymg/edit) and provide feedback in comments.
- [x] [Research plan](https://github.com/department-of-veterans-affairs/vagov-claim-classification/blob/main/collab-cycle/%5Bimproved%20autosuggest%20component%5D/research-plan.md) rcvd 3/22/2024; see latest version on [Google Docs](https://docs.google.com/document/d/1YoQdCVqATHc4Ps7oxQo4oOkOF5paI92Q4IHFyi72oqY/edit)
- [ ] https://github.com/department-of-veterans-affairs/va.gov-team/issues/76766
- [x] **Review prototype**  [Prototype in CodePen](https://codepen.io/ccteam/live/ZEZeMPR) rcvd 3/22/2024 audited 3/26/2024
- [X] **Review [live build](https://www.va.gov/disability/file-disability-claim-form-21-526ez/new-disabilities-revised/add)** (Second audit by Jamie)
- [X] [Request AT device and tech to be used in sessions](https://github.com/department-of-veterans-affairs/va.gov-team/issues/76707#:~:text=and%20Device%20Type.-,Table%202%3A%20Technology,-Toggle%20to%20view) - [Ref. Google Sheet - Scheduling Source of Truth](https://docs.google.com/spreadsheets/d/1nrqw-iekbkWM2oDwBPvaFqo76tOHZJNn55RTJQviixQ/edit?gid=0#gid=0&range=K1:K9)
- [x] [Receive AT device and tech from Team in research ticket](https://github.com/department-of-veterans-affairs/va.gov-team/issues/76707#:~:text=and%20Device%20Type.-,Table%202%3A%20Technology,-Toggle%20to%20view) to be used in sessions to prep for pilot = `JAWS + Windows`
- [X] **Run pilot** with VFS Researcher (prior to the start of research in "early" July 2024) - 7/22/2024
- [X] **During Pilot** Note that there are two variations**: "removing additional information" and "Adding additional information to component". "We can test variations of both options or consolidate to one if there's a strong opinion."
- [X] [**Note Taking**](https://docs.google.com/document/d/1aqAMiWe55lQTlnwKzs7VxtKCJGTQOh_TMnO9z4gjj_w/edit#heading=h.7b53y4dqihzk): **Pilot Prep and Notes** - Jamie and Sara
```

**Comment by sara-amanda** [comment]
## Timeline Shift

- April 2024 changed to May 2024
    - [See comments in the main research ticket for additional info. ](https://github.com/department-of-veterans-affairs/va.gov-team/issues/76707#issuecomment-2050109744)

- [x] This ticket title, high-level overview, body and tasks have been updated to reflect the change. 

@coforma-terry @coforma-jamie

**Comment by sara-amanda** [comment]
## Removed

> - **Team is aware of multiple studies:** 
>     - ⚠️ [Multiple Studies](https://github.com/department-of-veterans-affairs/va.gov-team/issues/76707#:~:text=Multiple%20Studies%20Week%20of%204/8/2024)
>     - 👍 [Team confirmed awareness of multiple studies 3/22/2024](https://github.com/department-of-veterans-affairs/va.gov-team/issues/76707#issuecomment-2016037133)

**Comment by sara-amanda** [comment]
## Timeline Shift

- May 2024 changed to June/July 2024
- [See comments in the main research ticket for additional info.](https://github.com/department-of-veterans-affairs/va.gov-team/issues/76707#issuecomment-2124837516)
- This ticket title, high-level overview, body and tasks have been updated to reflect the change.

@coforma-terry @coforma-jamie

**Comment by coforma-jamie** [comment]
Heard from the team: "Timing on research with AT users is TBD, but the earliest this could begin (starting with the pilot) is early July."

**Comment by sudeeptibh** [comment]
update - we will kick off recruiting next week with the goal of a 2-week recruitment period and to do the usability research week of July 22.

**Comment by sara-amanda** [comment]
## CAIA Support
[Thank you for the update!](https://github.com/department-of-veterans-affairs/va.gov-team/issues/76779#issuecomment-2207135062) @sudeeptibh 

### Pilot
- This ticket is for a pilot session. 
    - Are you still wanting to try for a pilot session with AT, prior to your research starting on 7/22, since we did the two audits for you? (See below section on Previous Audits Completed)
    - I am only mentioning this because the week of the 15th we do have limited capacity and two other research sessions going at that time. 
- I will cross-post the other information below in the main ticket. 😄 

### Previous Audits Completed
- Previously, @coforma-jamie completed an audit of the prototype in CodePen [ticket 76766](https://github.com/department-of-veterans-affairs/va.gov-team/issues/76766). 
- Additionally, @coforma-jamie completed an audit of the prototype in Staging [ticket 83767](https://github.com/department-of-veterans-affairs/va.gov-team/issues/83767)
- **Confirming**: Is this the URL that will be used in the testing sessions? 
    - URL: https://staging.va.gov/disability/file-disability-claim-form-21-526ez/new-disabilities-revised/add
User: Any user who can file a disability claim can get to this page for adding new disabilities. Allan Butler, Karl Howard, or Johnnie L. Weaver have worked fine for us.

### Number of Sessions
- **Confirming**: How many sessions are you looking to have covered? 
    - It looks like you will be needing support for two sessions, based on the research plan? 
    - I have updated our research outlook tracking with the two sessions for now. 
    - Just let us know when you have the dates/times and AT information available from Perigean. 

> At least 2 but no more than 3 Veterans who use assistive technology (AT) such as screen readers (AT/reader) or magnifiers (AT/magnifier)

## Capacity Update

We are currently experiencing limited capacity; however, your testing the week of 7/22, does not appear to have any conflict at this time for any current scheduled testing or PTO. 

cc: @coforma-jamie @EvanAtCoforma @SarahKay8 @NaomiPMC @artsymartha68

**Comment by coforma-jamie** [comment]
Spoke with team - I'll be piloting using Windows + JAWS.

URL: https://www.va.gov/disability/file-disability-claim-form-21-526ez/new-disabilities-revised/add

**Comment by sara-amanda** [comment]
## Pilot Session Takeaways

[Google Doc: Navigate to the Pilot Takeaways Section](https://docs.google.com/document/d/1aqAMiWe55lQTlnwKzs7VxtKCJGTQOh_TMnO9z4gjj_w/edit#heading=h.2iudajhvme4t)

- Logging in and getting to the URL may be a challenge. 
- Inform the user that they may need to wait, while the page loads re: “We are checking to see if you have an intent to file, and that could take a couple of minutes.”  That should read aloud to a screen reader user, so they know it is loading and why. 
    - ^Not their page, but they can share that with the team responsible. 
- We recommend allowing the user to explore the page first. 
- Error Messages: Tricky, when you get the error message and you try to remove it and the error message comes up when you try to add another condition. Content: In general the content will confuse users “Enter a condition” “enter your condition” it doesn’t come off as an error message as you are just hearing it. 
- Verbose: Tendonitis (tendinitis) in xxx (addressing spelling errors) 
- Bypassing with Free Text: As you type you get the pop-up menu and it will announce that there are three items in the pop-up menu. Now, if I am typing something and I know what it is and maybe it is just easier for me to type it and not go into the popup menu at all I can type the whole word, hit tab and it takes me to the save button and you can bypass the pop-up menu when typing in free text. Wondering if some users, this one wasn’t in the list, why not just save it? Free text isn’t being validated? Automation if they choose from the list.
- Alert - one question on a page, it is easy for someone to get down there, on something like this no one is going to get to the alert. Exception? Move the alert under add a condition or above back and continue, you won’t come across it otherwise.

**Comment by sara-amanda** [comment]
Thanks for including, CAIA A11Ys in your research pilot! We appreciate this opportunity to work with your team, to help make your product as accessible as possible. 😄 

- [x] Shared feedback
- [x] Completed session
- [x] Closing out ticket

cc: @NaomiPMC @coforma-jamie
